guides/micronaut-graalvm-native-image-google-cloud-platform-cloud-run/micronaut-graalvm-native-image-google-cloud-platform-cloud-run.adoc

```diff
@@ -0,0 +1,215 @@
+include::{commondir}/common-header.adoc[]
+
+You are going to deploy a https://www.graalvm.org/reference-manual/native-image/[GraalVM Native Image] of  Micronaut application to https://cloud.google.com/run[Google Cloud Run].
+
+> Develop and deploy highly scalable containerized applications on a fully managed serverless platform.
+
+include::{commondir}/common-requirements.adoc[]
+
+You need a https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account and a GCP project.
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-create-app.adoc[]
+
+=== Controller
+
+In order to create a microservice that responds with "Hello World" you first need a controller.
+
+Create a Controller:
+
+source:HelloController[]
+
+callout:controller[number=1,arg0=/hello]
+callout:get[number=2,arg0=index,arg1=/hello]
+callout:text-plain[3]
+<4> A String "Hello World" is returned as the result
+
+=== Test
+
+Create a test to verify that when you make a GET request to `/hello` you get `Hello World` as a response:
+
+test:HelloControllerTest[]
+
+callout:micronaut-test[1]
+callout:http-client[2]
+callout:http-request[3]
+
+include::{commondir}/common-testApp.adoc[]
+
+== Google Cloud Platform (CLI)
+
+You need a https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account. 
+
+You need a GCP project:
+
+image:gcp-project.png[]
+
+== Google Cloud CLI
+
+Install https://cloud.google.com/sdk[Google Cloud CLI].
+
+Login to your Google Cloud Platform:
+
+```bash
+> gcloud auth login 
+```
+
+Change your project: 
+
+```bash
+> gcloud config set project micronaut-guides
+```
```

Comment:
These are markdown, not asciidoc

Reply:
addressed via https://github.com/micronaut-projects/micronaut-guides/pull/560/commits/943d50d30ec030e7aa41f3a03b5368e89da8c8e0